### PR TITLE
feat: enable debug level logs support

### PR DESCRIPTION
**Pull request by surajgour-d11**
# Odin version(x.y.z)

### CODE REVIEW

File: internal/ui/logger.go [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+
 	log.SetFormatter(&log.TextFormatter{
-		ForceColors:   true,
-		DisableColors: false,
+		ForceColors:     true,
+		DisableColors:   false,
+		TimestampFormat: "2006-01-02 15:04:05", // Custom format
+		FullTimestamp:   true,
 	})
+	if os.Getenv(constant.LogLevelKey) == "yes" {
```
Comment (akshaypatidar1999) [inline internal/ui/logger.go:17]: Instead of taking `yes` take actual log level itself

Comment (surajgour-d11) [reply internal/ui/logger.go:17]: Then keeping `ODIN_LOG_LEVEL` as key

Comment (akshaypatidar1999) [reply internal/ui/logger.go:17]: Yes

File: internal/ui/logger.go [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+		DisableColors:   false,
+		TimestampFormat: "2006-01-02 15:04:05", // Custom format
+		FullTimestamp:   true,
 	})
+	level, err := log.ParseLevel(strings.ToLower(os.Getenv(constant.LogLevelKey)))
+	if err != nil {
+		log.SetLevel(level)
+	} else {
+		// log error message
+		log.Error("Invalid log level")
```
Comment (akshaypatidar1999) [inline internal/ui/logger.go:23]: `log.error` will not exit the command, use `log.Fatal`

Comment (surajgour-d11) [reply internal/ui/logger.go:23]: Setting incorrect log levels should not impact command flow. Maybe a warning message

Comment (akshaypatidar1999) [reply internal/ui/logger.go:23]: Yeah a warning message with fallback to `info` works

Comment (surajgour-d11) [reply internal/ui/logger.go]: done

**Review by akshaypatidar1999 — APPROVED** [review]

**Pull request merged**